app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -14,6 +14,19 @@ class VerifyCsrfToken extends BaseVerifier
     protected $except = [
         '/User/Settings/Ajax',
         '/Microblog/Comment/Show/*',
-        '/Forum/Comment/*'
+        '/Forum/Comment/*',
     ];
+
+    public function handle($request, \Closure $next)
```

Comment:
Szukałem jak to dobrze zrobić, ale nie udało mi się znaleźć nic lepszego niż modyfikacja middleware. Widać za słabo szukałem (google dla "middleware only subdomain" nie znalazło rano nic ciekawego).

Hm, to przerobię tak.